### PR TITLE
entry points with dynamic naming

### PR DESCRIPTION
**Pull request by deanshub**
### 🔦 Summary
entry points with dynamic naming, will allows multiple ooi widgets

### CODE REVIEW

File: packages/create-yoshi-app/templates/flow-editor/javascript/__tests__/viewerApp.e2e.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const { viewerUrl } = require('../dev/sites');
 
-describe('Viewer App', () => {
+describe.skip('Viewer App', () => {
```
Comment (ranyitz) [inline packages/create-yoshi-app/templates/flow-editor/javascript/__tests__/viewerApp.e2e.js]: Why is this skipped?

File: packages/create-yoshi-app/templates/flow-editor/javascript/{%packagejson%} [hunk 2]
```diff
@@ -11,10 +11,10 @@
     "start": "yoshi-flow-editor start --entry-point=dist/server.js",
     "precommit": "lint-staged",
     "pretest": "yoshi-flow-editor build",
-    "test": "yoshi test --jest",
+    "test": "yoshi-flow-editor test --jest",
```
Comment (ranyitz) [inline packages/create-yoshi-app/templates/flow-editor/javascript/{%packagejson%}]: `--jest` should be the default (so no need to add it)

File: packages/yoshi-flow-editor/config/wallaby-common.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+const { spawnSync } = require('child_process');
+const { isAbsolute } = require('path');
```
Comment (ranyitz) [inline packages/yoshi-flow-editor/config/wallaby-common.js]: Does this flow need a specific wallaby config? I think we can start without adding this concern. If we really want we can bring it from `yoshi-flow-legacy` (But I don't think it's important in this stage)

File: packages/yoshi-flow-editor/config/wallaby-jest.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+module.exports = function(wallaby) {
+  const wallabyCommon = require('./wallaby-common')(wallaby);
+  // We need to add a wallaby compiler, because apparently wallaby does not
```
Comment (ranyitz) [inline packages/yoshi-flow-editor/config/wallaby-jest.js]: Same as above (we can remove this file)

File: packages/yoshi-flow-editor/package.json [hunk 5]
```diff
@@ -26,6 +26,7 @@
     "yoshi-helpers": "4.22.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.7.4",
```
Comment (ranyitz) [inline packages/yoshi-flow-editor/package.json]: Why is this needed?

File: packages/create-yoshi-app/templates/flow-editor/javascript/__tests__/viewerApp.e2e.js (hunk 1 above)
Comment (deanshub) [reply packages/create-yoshi-app/templates/flow-editor/javascript/__tests__/viewerApp.e2e.js]: fixed in the next PR

File: packages/yoshi-flow-editor/package.json (hunk 5 above)
Comment (deanshub) [reply packages/yoshi-flow-editor/package.json]: don't need it no more 👍
